master/internal/resource_allocation.go

```diff
@@ -0,0 +1,45 @@
+package internal
+
+import (
+	"time"
+
+	"github.com/determined-ai/determined/master/internal/db"
+	"github.com/determined-ai/determined/master/pkg/actor"
+	"github.com/determined-ai/determined/master/pkg/actor/actors"
+)
+
+type aggregateTick struct{}
+
+type allocationAggregator struct {
+	db *db.PgDB
+}
+
+func (a *allocationAggregator) schedule(ctx *actor.Context) {
+	now := time.Now().UTC()
+	then := time.Date(now.Year(), now.Month(), now.Day(), 0, 1, 0, 0, time.UTC)
+	if then.Before(now) {
+		then = then.AddDate(0, 0, 1)
+	}
```

Comment:
maybe pull this out into something like `nearestDayStart` so it's immediately clear to the reader the intent?